src/payments-methods/express-helper/index.js

```diff
@@ -0,0 +1,51 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { decodeEntities } from '@wordpress/html-entities';
+import React from 'react';
+
+const { registerPaymentMethod, registerPaymentMethodExtensionCallbacks } = wc.wcBlocksRegistry;
+
+/**
+ * Internal dependencies
+ */
+import { PAYMENT_METHOD_NAME } from '../express/constants';
+import { AmazonComponent } from '../../utils';
+import { AmazonExpressContent, AmazonExpressLabel } from './payment-methods';
+import { settings } from '../express/settings';
+
+const label =
+	decodeEntities(settings.title) ||
+	__('Amazon Pay', 'woocommerce-gateway-amazon-payments-advanced');
+
+// Unset all other Gateways.
+if ( settings.allOtherGateways ) {
+	let hideAllOtherPaymentGateways = {};
+	for ( const offset in settings.allOtherGateways ) {
```

Comment:
Not blocker, but I think is a bit more readable like this:

	for ( const gateway of settings.allOtherGateways ) {
		hideAllOtherPaymentGateways[ gateway ] = () => { return false; };
	}

Reply:
Even though i am not sure why, settings.allOtherGateways is an object on JS side  while php returns a  non value => pair array.